tests/fv3_conf/regional_run.IN

```diff
@@ -1,9 +1,7 @@
-rsync -arv @[INPUTDATA_ROOT]/FV3_regional_input_data/. .
-
 rm -rf INPUT RESTART
 mkdir  INPUT RESTART
 
-rsync -arv @[INPUTDATA_ROOT]/fv3_regional_control/INPUT/. INPUT/.
+rsync -arv @[INPUTDATA_ROOT]/INPUT/. INPUT/.
```

Comment:
Why is this file changed if new test is using `regional_run_stoch.IN`?

Reply:
Sorry, Dusan.  I reverted this file in the original version in the latest commit.

Reply:
Sorry, @DusanJovic-NOAA, just reverted to original regional_run.IN in the latest commit.